Sources/Core/Pretty.swift

```diff
@@ -159,6 +119,43 @@ struct Pretty {
         }
     }
 
+    private func asValueString<T>(_ target: T, debug: Bool) -> String? {
+        // Note:
+        // The conditions for being a `ValueObject`:
+        // - has only one field
+        // - that field is `Premitive`
+
+        let mirror = Mirror(reflecting: target)
+
+        guard !debug, mirror.children.count == 1 else { return nil }
+
+        return mirror.children.first.flatMap { asPremitiveString($0.value, debug: debug) }
+    }
+
+    private func asPremitiveString<T>(_ target: T, debug: Bool) -> String? {
+        switch target {
+        case let value as String:
+            return #""\#(value)""#
+
+        case let value as Int:
+            return "\(value)"
+
+        case let value as Bool:
+            return "\(value)"
+
+        case let url as URL:
+            if debug {
+                return #"URL("\#(url.absoluteString)")"#
+            } else {
+                return url.absoluteString
+            }
+
+        default:
+            // TODO: support other premitive type that not has child
```

Comment:
We should support other type such as `Double`, `Range`, `Date` and others maybe.